magda-web-client/src/Components/Dataset/Add/DatasetAddFilesPage.tsx

```diff
@@ -89,102 +89,98 @@ class DatasetAddFilesPage extends React.Component<
                 // --- skip the directory item
                 continue;
             }
+            if (!thisFile) continue;
```

Comment:
The changes in this file are simply de-indenting the code as this inverted `if` statement lets us do that.

Reply:
Fair enough. Fixed.